plugins/bundle/plugin_test.go

```diff
@@ -2082,6 +2203,19 @@ func getTestSignedBundle(t *testing.T) bundle.Bundle {
 	return b
 }
 
+func getTestRawBundle(t *testing.T) io.Reader {
+	t.Helper()
+
+	b := getTestBundle(t)
+
+	var buf bytes.Buffer
+	if err := bundle.NewWriter(&buf).UseModulePath(true).Write(b); err != nil {
+		t.Fatal("unexpected error:", err)
+	}
+
+	return &buf
+}
```

Comment:
Looks like we could use this helper a bit more often above? Or am I missing something?

Reply:
Ignore me -- I thought this only did
```go
bundle.NewWriter(&amp;buf).UseModulePath(true).Write(b)
```
in a helper, but it's using `getTestBundle`, and wasn't meant to replace the similar calls above. We're good here. :D